src/switch/switch.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let trackClass = null;
+  export let thumbClass = null;
+
+  export let value = false;
+  export let disabled = false;
+  export let slotLeft = false;
```

Comment:
I feel like the slot will almost always be used just for a label, so I feel like the name `labelLeft` would be more appropriate.

Reply:
This is kind of for consistency with Checkbox and RadioButton, which also have a `slotLeft` prop, while CheckboxGroup has `labelsLeft`. In that way, the Checkbox and RadioButton components are meant to be used sparingly, where extra customization is needed, and in that case the `slotLeft` name wouldn't scare people away. As for the switch, again, consistency.